packages/ledger/src/lib/ledger.spec.ts

```diff
@@ -78,51 +101,101 @@ afterEach(() => {
+  it("returns account objects", async () => {
+    const accountId = "amirsaran.testnet";
     const { wallet } = await createLedgerWallet();
     await wallet.connect({
-      derivationPaths: ["derivationPath"],
+      derivationPaths: ["44'/397'/0'/0'/1'"],
     });
     const result = await wallet.getAccounts();
     expect(result).toEqual([{ accountId }]);
   });
+  it("returns empty list because not connected", async () => {
+    const { wallet } = await createLedgerWallet();
+    const result = await wallet.getAccounts();
+    expect(result).toEqual([]);
+  });
+});
+
+describe("signAndSendTransaction", () => {
+  it("signs and sends transaction", async () => {
+    const { wallet, ledgerClient } = await createLedgerWallet();
+    await wallet.connect({
+      derivationPaths: ["44'/397'/0'/0'/1'"],
+    });
+    const result = await wallet.signAndSendTransaction({
+      signerId: "amirsaran.testnet",
+      receiverId: "guest-book.testnet",
+      actions: [],
+    });
+    expect(ledgerClient.sign).toHaveBeenCalled();
+    expect(result).toEqual(getSendTransactionResponse());
+  });
 });
 
-// describe("signAndSendTransaction", () => {
-//   it("signs and sends transaction", async () => {
-//     const { wallet, authData } = await createLedgerWallet();
-//     await wallet.connect({
-//       accountId: authData.accountId,
-//       derivationPaths: [authData.derivationPath],
-//     });
-//     const result = await wallet.signAndSendTransaction({
-//       signerId: "amirsaran.testnet",
-//       receiverId: "guest-book.testnet",
-//       actions: [],
-//     });
-//     expect(result).toEqual(null);
-//   });
-// });
+describe("signAndSendTransactions", () => {
+  it("signs and sends only one transaction", async () => {
+    const { wallet, ledgerClient } = await createLedgerWallet();
+    await wallet.connect({
+      derivationPaths: ["44'/397'/0'/0'/1'"],
+    });
+    const transactions: Array<Transaction> = [
+      {
+        signerId: "amirsaran.testnet",
+        receiverId: "guest-book.testnet",
+        actions: [],
+      },
+    ];
+    const result = await wallet.signAndSendTransactions({
+      transactions: transactions,
+    });
+    expect(ledgerClient.sign).toHaveBeenCalled();
+    expect(result).toEqual([getSendTransactionResponse()]);
+  });
+
+  it("signs and sends multiple transactions", async () => {
+    const { wallet, ledgerClient } = await createLedgerWallet();
+    await wallet.connect({
+      derivationPaths: ["44'/397'/0'/0'/1'"],
+    });
+    const transactions: Array<Transaction> = [
+      {
+        signerId: "amirsaran.testnet",
+        receiverId: "guest-book.testnet",
+        actions: [],
+      },
+      {
+        signerId: "amirsaran.testnet",
+        receiverId: "guest-book.testnet",
+        actions: [],
+      },
+    ];
+    const result = await wallet.signAndSendTransactions({
+      transactions,
+    });
+    expect(ledgerClient.sign).toHaveBeenCalled();
```

Comment:
We could improve this slightly by checking it's called `transactions.length` times. Similarly, we could assert `result` returns the same length too (without asserting the values since TypeScript has us covered for this)